### ISSUE TITLE
"Copy invite link" seems to be not working.

### ISSUE THREAD

**Issue by hasanfalizada** [mattermost/mattermost#20432]
#### Summary
After "Copy invite link" button click in the "Invite members" dialog invite link is not being copied to the clipboard. Fresh Docker deployment.

#### Steps to reproduce
Click on the "Invite members" button on the side panel, then click the appropriate button in the opened dialog.

Mattermost Enterprise Edition
Modern communication from behind your firewall.

Mattermost Version: 6.3.8
Database Schema Version: 6.3.8
Database: postgres

#### Expected behavior
Invite link copied to the clipboard.

#### Observed behavior (that appears unintentional) 
Literally nothing. No log entries for the event.

![image](https://user-images.githubusercontent.com/18369497/173025169-9767f91c-de38-4d8e-98be-3d085fd0b076.png)

**Comment by hasanfalizada** [comment]
This behavior is observed on deployment without NGINX (HTTP).
HTTPS is ok.

**Comment by amyblais** [comment]
@hasanfalizada I wasn't able to reproduce this on 6.3.9. Is this setting set to "true" https://docs.mattermost.com/configure/configuration-settings.html#enable-account-creation?

**Comment by hasanfalizada** [comment]
Yes.
![image](https://user-images.githubusercontent.com/18369497/173176407-a0724cb3-3285-4a63-bc9d-8e47d7597fb6.png)

Firstly, I thought that there was a problem with my deployment, so I run a preview container without NGINX and the result was the same.
I've checked this behavior in different browsers, also.
But if run with NGINX - everything works.

**Comment by gustavosbarreto** [comment]
The same here

**Comment by amyblais** [comment]
The ticket was closed since there is no solution or workaround for this specially for the desktop app. Following are the technical reasons. Pasting below comments from the developers:

"It is true that navigator.clipboard is not available for non-secure sites or for localhost, the only and widely used workaround is to use document.execCommand, (which is a deprecated api by several browsers due to the potential security risks it has), and it  is already used in our code base as a fallback for the original navigator.clipboard unavailability. 
The fact is that Electron uses Chromium as the running app browser and  Chromium has already deprecated the document.execCommand, so clipboard is not available because the lack of certificate and document.execCommand is deprecated meaning there is no way to make it work.

TL;DR: for non https or localhost desktop app, the copy link button won’t work and there is nothing technically possible to do about this. Only option is that the customer uses a ssl certificate, even a self-signed certificate would work."

**Comment by amyblais** [comment]
I've opened a docs issue to take a look at adding the details to the docs https://github.com/mattermost/docs/issues/6499.
